buildSrc/src/main/groovy/MavenToolchain.groovy

```diff
@@ -26,11 +26,11 @@ class MavenToolchain {
 
   private static def exe = OperatingSystem.current().isWindows() ? '.exe' : ''
 
-  static def javaHome = { v -> 
+  static def javaHome = { v ->
```

Comment:
Although we have a check style rule ... that seems to no apply to build files :wink: so I am fine with that one staying in.